cluster/stone-prd-rh01/tenants/rhtap-migration-tenant/hac-standard.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: hac-standard
```

Comment:
hac is the name of just one team. consoledot-frontend-standard might be a better name for the policy